content/cloud-docs/run/states.mdx

```diff
@@ -46,7 +46,29 @@ _Leaving this stage:_
 
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
-## 3. The Cost Estimation Stage
+## 3. The Post-Plan Stage
+
+This phase only occurs if there are [run tasks](/cloud-docs/workspaces/settings/run-tasks) enabled
+for the workspace, and it executes any configured run tasks after the Plan has completed. This phase
+is entered for all runs, including [speculative](/cloud-docs/run/#speculative-plans) runs. During
+this phase, Terraform Cloud sends information about the run to the configured external system and
+waits for a `passed` or `failed` response to determine whether the run can continue.
+
+-> **Note:** The information sent to the configured external system includes the [JSON output](/internals/json-format) of your plan.
+
+_States in this stage:_
+
+- **Running tasks:** Terraform Cloud is currently waiting for a response from the configured external system(s).
+  - External systems must respond initially with a `200 OK` acknowledging the request is in progress. After that, they have 10 minutes to return a status of `passed` or `failed`, or the timeout will expire and the task will be assumed to be in the `failed` status.
+
+_Leaving this stage:_
```

Comment:
@laurapacilio it would be great to get your input on this section. I'm not sure if _Advisory_ and _Mandatory_ should be treated as proper nouns and capitalized. I think we have had this discussion before, but my memory is failing me 😆

Reply:
ahhh I think our decision was that we would not capitalize this, and that matches the construction we have on this page as well: https://www.terraform.io/cloud-docs/workspaces/settings/run-tasks

To maintain consistency, let's keep it not capitalized here. We generally do not capitalize feature names, etc. in our docs. It is tempting because this is technically a task type, but it's also used as an adjective in this sentence. Because of its usage here and the points I made above, I think it's appropriate to keep it uncapitalized. However, I'm going to check in with the rest of the tech writers on this at our next meeting and get some formal guidance developed. Once that's ready, I will take the responsibility of coming back into these docs and updating this if what we decide on is different than the guidance I'm giving you now!